java-frontend/src/test/java/org/sonar/java/resolve/SymbolTableTest.java

```diff
@@ -1334,4 +1335,11 @@ public void double_lambda_type_propagation() {
     JavaSymbol my = result.symbol("my");
     assertThat(my.usages()).hasSize(2);
   }
+
+  @Test
+  public void conditional_expression_in_lambda() {
+    Result result = Result.createFor("ConditionalExpressionInLambda");
+    JavaSymbol foo = result.symbol("foo");
+    assertThat(foo.usages()).hasSize(1);
```

Comment:
Please make a direct test : this one is indirect. You should test the expected type of the conditional expression.